coops-ui/src/components/Dashboard/TodoList.vue

```diff
@@ -5,6 +5,11 @@
       attach="#todo-list"
     />
 
+    <ConfirmDialog
+      ref="confirmCancelPaymentDialog"
+      attach="#todo-list" data-test-id="cancel-payment-confirm-dialog"
```

Comment:
Please add a `data-test-id` to the other instance of this dialog (above) to be able to select one or the other.

Reply:
FYI, adding `data-test-id` here can be used to select the stub div for the dialog (which contains nothing).
Instead, adding `content-class` to the v-dialog can be used to select the actual div that contains the dialog elements.
Based on this, I think adding data-test-id here is not useful.